### ISSUE TITLE
"html.cke_copyformatting_active" visible in Styles Dropdown

### ISSUE THREAD

**Issue by djamell** [w8tcha/dnnckeditor#39]
Running CKEditor Provider 2.01.16 on DNN 8.0.4.

At the bottom of the "Styles" Dropdown there is a string (html.cke_copyformatting_active) that should not be there.  See the attached image:

![copyformatting](https://user-images.githubusercontent.com/312711/43645455-8962bd10-96f7-11e8-82b4-86311f7cfe4f.jpg)

It appears in all browsers except Firefox.

How do I remove this?

**Comment by w8tcha** [comment]
What content did you enter in the config tab under StylesSet?

**Comment by djamell** [comment]
This is what I have for StylesSet:

```
[
// Inline Styles
{ name: 'Red Text', element: 'span', attributes: { 'class': 'RedText'} },
{ name: 'Blue Text', element: 'span', attributes: { 'class': 'BlueText'} },
{ name: 'Green Text', element: 'span', attributes: { 'class': 'GreenText'} },
{ name: 'Oswald Font', element: 'span', attributes: { 'class': 'Oswald'} },
{ name: 'Small Text', element: 'span', attributes: { 'class': 'small'} },
{ name: 'Big Text', element: 'span', attributes: { 'class': 'big'} },
	
// Object Styles
{ name: 'Image Fluid Lt', element: 'img', attributes: { 'class': 'image-fluidlt'} },
{ name: 'Image Fluid Rt', element: 'img', attributes: { 'class': 'image-fluidrt'} },
{ name: 'Image Fluid', element: 'img', attributes: { 'class': 'img-responsive'} },
]
```

Also, the string doesn't appear in Firefox, but does in Chrome, IE, and Edge browsers.

**Comment by w8tcha** [comment]
Fixed in 2.01.17
